src/components/MainPage/HotPlaces.jsx

```
@@ -40,39 +41,17 @@ const HotPlaces = ({ accessToken, refreshAccessToken }) => {
-          setError("로그인이 필요합니다.");
-        }
+      const response = await axios.get(`https://meong9.store${selectedApiPath}`, { headers });
+      if (response.data && response.data.data) {
+        setPlaces(response.data.data);
       } else {
-        console.error("데이터를 가져오는데 실패했습니다:", error.message);
-        setError("데이터를 가져오는데 실패했습니다.");
+        setPlaces([]);
       }
+    } catch (error) {
+      setError("데이터를 가져오는데 실패했습니다.");
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)